### ISSUE TITLE
Taxonomy categories not displaying

### ISSUE THREAD

**Issue by dlschwartz** [srophe/srophe-eXist-app#1104]
@wsalesky With the push of the newly transformed taxonomy TEI records to dev, the categories tab has stopped working: http://wwwb.library.vanderbilt.edu/exist/apps/srophe/taxonomy/browse.html?view=categories. It only lists languages and then when you select that only three of the 27 languages in the taxonomy appear. Interestingly, things seem to be working correctly in the advanced browse tab (my earlier concern about a problem with the transform from spreadsheet to TEI was a mistake!). Perhaps there was some kind of a problem with indexing? Thanks Winona.

**Comment by wsalesky** [comment]
The browse categories tab uses `tei:entryFree[@subtype='category']` this does not seem to be in the updated version of the data. Should I use the `tei:relation[ref="skos:broadMatch"]/@passive` instead?

**Comment by dlschwartz** [comment]
Of course, now I remember adding those in manually after the fact. I'll do that again and see how things go. Then I'll add that to the spreadsheet/xslt. 

I trust the RDF generation wasn't looking at the subtype="category". It seems like the skos:broadMatch would be preferable and the subtype="category" was solely for the categories tab. Please correct me if I'm wrong about that.

Thanks Winona.

**Comment by wsalesky** [comment]
That is correct.

**Comment by dlschwartz** [comment]
I think this is all fixed (although I clearly have some work yet to do on my xslt!) Thanks Winona.

**Comment by dlschwartz** [comment]
@wsalesky There seems to be an additional problem I can't figure out. The taxonomy categories now appear which is why I closed this issue. However, when you select one of the categories and see all of the keywords nested inside of it, the URI's/links are for the category, not for the nested keyword. See http://wwwb.library.vanderbilt.edu/exist/apps/srophe/keyword/administration. 

I spot-checked the new version of a couple of the taxonomy xml files against the older version (somewhat randomly I chose this commit: https://github.com/srophe/srophe-app-data/commit/0bf3b7cee9a8eacd4292a8b63d6dafc7ab5ac82b). What I understand to be the relevant portions of the xml documents do not differ. 

Am I missing something? Thanks Winona.
